Templates/Biostats_template.Rmd

```diff
@@ -30,6 +72,74 @@ output:
 ---
 ```
 
+Add this chunk to load libraries, define options and source the figure setting file.
+!!! HOW CAN I DISPLAY THE FULL CHUNK? !!!
+```{r setup-template, eval=FALSE, include=TRUE}
+#add all libraries that need loading
+library(learnr)
+
+#define options
+knitr::opts_chunk$set(echo = TRUE)
+
+#source figure settings
+source("set_theme_file.R")
+```
+
+
+
+# Text size, colours, etc
+
+Don't worry, the css styleguide file will take care of this.
+
+
+# Figures
+
+The set_theme_file.R will take care of colours, text size, theme etc in all the figures.
+This file has to be sourced in each Rmarkdown file (see Starting an .Rmd file above).
+
+The figures have the following settings:
+- theme_minimal()
+- text size: 12
+- colour: ... for discrete data use... and for continouse data use ...
+
+
+Each figure will have two chunks of code:
+- a chunk that displays the code, but does not do anything
+- a chunk that makes and prints the figure with the correct settings
```

Comment:
Might not need this with the solution for changing default scales. Would be nice to do without. 
If we do have to use this two chunk strategy it should be something like this

Chunk 1

 '''{r my-fig-prep, echo  = TRUE, eval = TRUE, results='hide'}
my_plot <- ggplot(penguins, aes(...)) + 
geom_point()
my_plot
'''
chunk 2
'''{r print-my-figure, echo = FALSE, eval = TRUE, fig.cap = "my fig cap"}
my_plot + scale_*_*() 
'''